src/MessageManager.h

```diff
@@ -92,6 +92,7 @@ struct Message
 	explicit Message( const RString &s );
 	explicit Message(const MessageID id);
 	Message( const RString &s, const LuaReference &params );
+	Message(Message&& other);
```

Comment:
NewField uses create_did_message to abstract the common  part of the Message that did_tap_note and did_hold_note use.  The move constructor is supposed to be more efficient than the copy constructor.